init.coffee

```diff
@@ -13,11 +13,30 @@ exports.configure = [
     message: 'Do you need netlify hosting?'
   },
   {
+    type: 'list'
+    name: 'css'
+    message: 'What would you like to use as your CSS Preprocessor?'
+    choices: ['postcss', 'stylus']
```

Comment:
yeah, i know it's weird right now and you can use plugins for each to do the same things, but think of it as a slow transition to postcss - but we still depend on a lot from stylus, like axis.